index.html

```diff
@@ -6,9 +6,158 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Homework 6</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css ">
+    <link rel="stylesheet" href="./lifehack-sans-regular/Lifehack Sans Regular/Lifehack Sans Regular.ttf">
+    <link rel="stylesheet" href="./all.css">
     <link rel="stylesheet" href="./style.css">
 </head>
 <body>
+<div class="header">
+    <div class="header-logo">
+        <img src="./menu/KIDS-jump-TECH-2 2.png" alt="logo">
+    </div>
+    <div class="header-menu">
+        <div class="header-menu-items">Products</div>
+        <div class="header-menu-items">Test drive</div>
+        <div class="header-menu-items">For kids</div>
+        <div class="header-menu-items">Games</div>
+        <div class="header-menu-items">Contacts us</div>
+        <div class="header-menu-items">Faq</div>
+    </div>
+    <div class="header-title">
+        <div class="title">GAMES</div>
+        <div class="subtitle">More than 90 educational games and applications are included with the delivery of the interactive floor.</div>
+    </div>
+    <div class="header-img">
+        <img class="header-img-group" src="./menu/group.png" alt="200+">
+        <img class="header-img-horse" src="./menu/horse.png" alt="horse">
+        <img class="header-img-knowledge" src="./menu/knowledge.png" alt="knowledge">
+    </div>
+    <div class="header-tag">
```

Comment:
rework it, it shouldn't be in the `header`
move it to the `body`
add right spaces, because now it is closer to the left than should be
and add the right styles to the tag, because now it's not similar to the design